### PR TITLE
kubeadm: add document for upgrading from 1.13 to 1.14 (single CP and HA)

### PR DESCRIPTION
**Pull request by neolit123**
+ remove doc for upgrading 1.10 -> 1.11

we merged the HA and regular upgrade docs.

follow up from https://github.com/kubernetes/website/pull/12919

xref https://github.com/kubernetes/kubeadm/issues/1422

@kubernetes/sig-cluster-lifecycle-pr-reviews 
/priority critical-urgent
/assign @jimangel @fabriziopandini @timothysc

### CODE REVIEW

**Comment by neolit123** [comment]
we merged the HA and regular upgrade docs.
@detiber PTAL if you have some time.

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+
+[swap]: https://serverfault.com/questions/684771/best-way-to-disable-swap-in-linux
+### Additional information
+
+- All containers are restarted after upgrade, because the container spec hash value is changed.
+- You can upgrade only from one minor version to the next minor version.
+  That is, you cannot skip versions when you upgrade.
+  For example, you can upgrade only from 1.y to 1.y+1, not from 1.y to 1.y+2
+
+{{< warning >}}
```
Comment (fabriziopandini) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]: This warning can be removed

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+## Ugrade worker nodes
+
+The upgrade procedure on worker nodes should be executed one node at a time or few nodes at a time,
+without compromising the minimum required capacity for running your workloads.
+
+### Upgrade kubeadm
+
+1.  Upgrade kubeadm on all worker nodes:
+
+    {{< tabs name="k8s_install_kubeadm" >}}
```
Comment (fabriziopandini) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]: It seems this tab is not rendering well in preview 🤔

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+    kubectl drain ip-172-31-85-18
+    node "ip-172-31-85-18" cordoned
+    error: unable to drain node "ip-172-31-85-18", aborting command...
+
+    There are pending nodes to be drained:
+    ip-172-31-85-18
+    error: DaemonSet-managed pods (use --ignore-daemonsets to ignore): calico-node-5798d, kube-proxy-thjp9
+    ```
+
+## Upgrade the kubelet config
```
Comment (fabriziopandini) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]: This should be header 3

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+
+1. Restart the kubelet
+
+    ```shell
+    sudo systemctl restart kubelet
+    ```
+
+## Upgrade additional control plane nodes
+
+1.  Same as the first control plane node but use:
```
Comment (detiber) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]: Should it also be mentioned that there is no need to run `sudo kubeadm upgrade plan` as well?

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md [hunk 5]
```diff
@@ -0,0 +1,373 @@
...
+## Recovering from a failure state
+
+If `kubeadm upgrade` fails and does not roll back, for example because of an unexpected shutdown during execution, you can run `kubeadm upgrade` again.
+This command is idempotent and eventually makes sure that the actual state is the desired state you declare.
+
+To recover from a bad state, you can also run `kubeadm upgrade --force` without changing the version that your cluster is running.
+
+## How it works
+
+`kubeadm upgrade apply` does the following:
```
Comment (detiber) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]: Would it also be good to describe the steps that `kubeadm upgrade node experimental controlplane` performs as well?

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md [hunk 6]
```diff
@@ -0,0 +1,380 @@
...
+    {{< /tabs >}}
+
+### Cordon the node
+
+1.  Prepare the node for maintenance by marking it unschedulable and evicting the workloads. Run:
+
+    ```shell
+    kubectl drain $NODE --ignore-daemonsets
+    ```
+
```
Comment (DanyC97) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]: can you please add `The output is similar to this:` or `You should see output similar to this:` how you used already in this doc? this should align with style guidelines

Comment (DanyC97) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]: @neolit123 fyi ^^

Comment (neolit123) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]: updated.

**Comment by DanyC97** [comment]
/lgtm

thank you @neolit123

**Comment by timothysc** [comment]
/lgtm 
/approve 

@kubernetes/sig-docs-pr-reviews

**Pull request merged**